patterns/template-archive-writer.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Title: Writer Archive Template
+ * Title: Writer Archive
```

Comment:
We renamed the writer one for another pattern, are we keeping the name for these ones?

Reply:
If we merge this in before the other PR it's fine, then it will be overwritten by the following PR.